facility-manager.html

```diff
@@ -120,12 +121,15 @@
     like to any user's data, in any group (except annotate in Private groups)
     and Group Owners can do anything to any user's data in the groups they
     own.
+    <a href="https://docs.openmicroscopy.org/omero/5.3.3/sysadmins/admins-with-restricted-privileges.html" class="external-url">
```

Comment:
use ``latest/omero``

Reply:
Would http://docs.openmicroscopy.org/latest/omero5.3 be safer?

Reply:
You mean 5.4 since it is introduced in the coming release 
Probably safer

Reply:
I would have thought ```latest/omero/``` would be best, since this will be valid after 5.4.0.  Either way it will be 404 until release, so that's not a reason to use ```/5.4.0/```